src/sentry/conf/server.py

```diff
@@ -1682,6 +1686,17 @@ def create_partitioned_queues(name):
         "only_if": lambda settings, options: settings.SENTRY_USE_RELAY,
         "with_devserver": True,
     },
+    "chartcuterie": {
+        "image": "us.gcr.io/sentryio/chartcuterie:nightly",
+        "pull": True,
+        "volumes": {CHARTCUTERIE_CONFIG_DIR: {"bind": "/etc/chartcuterie"}},
+        "environment": {
+            "CHARTCUTERIE_CONFIG": "/etc/chartcuterie/config.js",
+            "CHARTCUTERIE_CONFIG_POLLING": "true",
+        },
+        "ports": {"9090/tcp": 7901},
+        "only_if": lambda settings, options: options.get("chart-rendering.enabled"),
```

Comment:
nit: I feel like if this is only for the devservice I prefer using a setting instead of options but 🤷

Reply:
This is not only for dev services. It will also be used to turn on and off this service as a whole.

Reply:
Ahhh, makes sense :+1: